### PR TITLE
Stationkeep

### PR DESCRIPTION
**Pull request by jkulik11**
Merging Gabe's stationkeeping and continuous thrust code with master after documentation additions and a testing white list addition.

### CODE REVIEW

File: EXOSIMS/Prototypes/Observatory.py [hunk 1]
```diff
@@ -789,8 +789,13 @@ def star_angularSep(self,TL,old_sInd,sInds,currentTime):
             # finding sign of angle
             u2_Az = np.dot(a3,u_new.T)
             sgn = np.sign(u2_Az)
+            sgn[np.where(sgn==0)] = 1
             sd = sgn*sd  #The star angular separation can be negative because it is with respect to a frame
-        
+            
+            #need to fix this, if angle is 180
+            oneEighty = np.where(sd.value - np.pi == 0)[0]
```
[the remark this answers is not in the document]
Comment (jkulik11) [reply EXOSIMS/Prototypes/Observatory.py:796]: I have no idea here. It makes sense to me to switch to the direct comparison, but I am having trouble understanding the signing convention for this angle... 

The angle changing is strange... it only changes signs for every other instance in which the angle is 180 degrees since the variable i is the index in the list of indices that give rise to 180 degree values.

[the remark this answers is not in the document]
Comment (dsavransky) [reply EXOSIMS/Prototypes/Observatory.py:796]: I think some more debugging is in order.  Quick check against direct comparison to confirm whether or not truncation error is an issue here, and then a deeper test to understand the indexing behavior. @gjsoto any comments?

[the remark this answers is not in the document]
Comment (gjsoto) [reply EXOSIMS/Prototypes/Observatory.py:796]: so it seems I added this in commit f920985 on 9-23-2019 on this branch, and I'm trying to remember why (also, just learned about 'git blame' and it's pretty handy). I *think* this had something to do with my FakeCatalog maps, but I forget the context exactly.  I at some point kept getting star separations piling up at 180deg and for aesthetics I pulled that stunt to alternate between 180 and -180 on my plots. At least from my commit message, it seemed the purpose was purely to alternate between positive and negative angles at 180deg separation. 

**Bottom Line:**  I believe that lines 795-798 are unnecessary, I think I added those so I could make some sort of plot look nicer. Don't think I meant for that to make its way to the master branch. Sorry for the confusion!

[the remark this answers is not in the document]
Comment (jkulik11) [reply EXOSIMS/Prototypes/Observatory.py:796]: Thank you so much Gabe! @gjsoto
